packages/data-point/lib/entity-types/entity-request/__snapshots__/resolve.test.js.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`resolveRequest log errors when request fails 1`] = `
+"404 - \\"not found\\"
+
+Entity info:
+  Id: test:test
+  - options: { json: true, url: 'http://remote.test/source1' }
+  - value: 'foo'
+
+  Request:
+  - error: 'not found'
+  - message: '404 - \\"not found\\"'
+  - statusCode: 404
+  - options: { json: true,
+  url: 'http://remote.test/source1',
+  callback: [Function: RP$callback],
+  transform: undefined,
+  simple: true,
+  resolveWithFullResponse: false,
+  transform2xxOnly: false }
```

Comment:
Similar comment to above:

```javascript
Request:
  Error: 'not found'
  Message: '404 - \\"not found\\"'
  StatusCode: 404
  Options: {
    json: true,
    url: 'http://remote.test/source1',
    callback: [Function: RP$callback],
    transform: undefined,
    simple: true,
    resolveWithFullResponse: false,
    transform2xxOnly: false
  }
```

Reply:
yup, would have to use another package for formatting, which ATM I dont think we can justify, certainly something to add later

Reply:
one I have an eye on is https://www.npmjs.com/package/print

Reply:
At least having the bullets is a little clearer. Thanks for making that change